### ISSUE TITLE
Mistake in files generation (on linux)

### ISSUE THREAD

**Issue by framp** [edorivai/nock-record#21]
After following the tutorial, nock-record is generating a file in almost the correct path.

It seems to be prepending `file:` before the path
![Screenshot_20200530_110153](https://user-images.githubusercontent.com/611109/83325602-49191a80-a265-11ea-9863-34d032642ea4.png)

This is happening because the stack trace on linux seems to include the `file://` protocol at the beginning of the path

**Comment by edorivai** [comment]
Hmm interesting. I'm running Linux as well (Ubuntu 19.10 at the moment), but am not running into this issue :thinking: .

Would you mind running the tests of this repo? 
```bash
git clone git@github.com:edorivai/nock-record.git
cd nock-record
yarn # or npm i, whatever
yarn test # or npm test
```

**Comment by framp** [comment]
tests are passing - I assume because it's not generating a file or checking what value it's reading

I inspected what stack-trace is returning: the first 2 lines the library, the third one is the test file with the file:// protocol prefixed.

With the code in https://github.com/edorivai/nock-record/pull/22 it works as expected and creates a file in the right place

**Comment by edorivai** [comment]
Fixed in #22, thanks @framp
